homeassistant/components/netatmo/webhook.py

```diff
@@ -0,0 +1,64 @@
+"""The Netatmo integration."""
+import logging
+
+from .const import (
+    ATTR_EVENT_TYPE,
+    ATTR_FACE_URL,
+    ATTR_ID,
+    ATTR_IS_KNOWN,
+    ATTR_NAME,
+    ATTR_PERSONS,
+    DATA_PERSONS,
+    DEFAULT_PERSON,
+    DOMAIN,
+    NETATMO_EVENT,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def handle_webhook(hass, webhook_id, request):
+    """Handle webhook callback."""
+    try:
+        data = await request.json()
+    except ValueError:
+        return None
+
+    _LOGGER.debug("Got webhook data: %s", data)
+
+    event_type = data.get(ATTR_EVENT_TYPE)
+
+    if event_type == "outdoor":
+        hass.bus.async_fire(
+            event_type=NETATMO_EVENT, event_data={"type": event_type, "data": data}
+        )
+        for event_data in data.get("event_list"):
+            evaluate_event(hass, event_data)
+    else:
+        evaluate_event(hass, data)
+
+
+def evaluate_event(hass, event_data):
```

Comment:
Please mark this as `@callback` and prefix the name with `async_` 
```suggestion
@callback
def async_evaluate_event(hass, event_data):
```